controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/ApplicationSerializer.java

```diff
@@ -320,6 +324,21 @@ private void toSlime(Map<HostName, RotationStatus> rotationStatus, Cursor array)
         });
     }
 
+    private void rotationsToSlime(List<AssignedRotation> rotations, Cursor parent, String fieldName) {
+        final var rotationsArray = parent.setArray(fieldName);
+        rotations.forEach(rot -> rotationsArray.addString(rot.rotationId().asString()));
+    }
+
+    private void assignedRotationsToSlime(List<AssignedRotation> rotations, Cursor parent, String fieldName) {
+        final var rotationsArray = parent.setArray(fieldName);
+        for (var rotation : rotations) {
+            final var object = rotationsArray.addObject();
+            object.setString("endpoint", rotation.endpointId().id());
```

Comment:
I think it would be better to use the same name as the `AssignedRotation` field. E.g. "endpoint" -> "endpointId". This and the corresponding `fromSlime` method should use constants for the field names.

Reply:
Agreed and done. I didn't do it this way because that was not how it was done for other values in this file.